config/01-ce-collector-audit-payload.conf

```diff
@@ -0,0 +1,20 @@
+# Settings for the collector audit payloads function.
```

Comment:
I can't drop the file, but do you mean I should move it to /usr/share/condor-ce/config.d (and change the comments accordingly)?